File: core/src/com/ovt/quest/screens/MinigamesScreen.java
package com.ovt.quest.screens;

import com.badlogic.gdx.Screen;
import com.badlogic.gdx.scenes.scene2d.ui.Table;
import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
import com.ovt.quest.QuestGame;

/**
 * Created by nikolay on 05/01/2018.
 */

public class MinigamesScreen implements Screen {

    private QuestGame game;

    public MinigamesScreen(QuestGame game) {
        this.game = game;
    }

    @Override
    public void show() {
        Table table = new Table();
        table.setDebug(true);
        table.setFillParent(true);

    }

    @Override
    public void render(float delta) {

    }

    @Override
    public void resize(int width, int height) {

    }

    @Override
    public void pause() {

    }

    @Override
    public void resume() {

    }

    @Override
    public void hide() {

    }

    @Override
    public void dispose() {

    }
}

File: core/src/com/ovt/quest/main_menu_screens/MinigamesScreen.kt
package com.ovt.quest.main_menu_screens

import com.badlogic.gdx.Gdx
import com.badlogic.gdx.Screen
import com.badlogic.gdx.graphics.GL20
import com.ovt.quest.QuestGame
import com.ovt.quest.archery.ArcheryScreen
import com.ovt.quest.commons.addClickListener
import com.ovt.quest.main_menu_screens.layout.MinigamesStage
import com.ovt.quest.three_in_a_row.ThreeInARowScreen

/**
 * Created by nikolay on 14.03.18.
 */
class MinigamesScreen(private val game: QuestGame) : Screen {

    private val stage = MinigamesStage(game)


    override fun show() {
        Gdx.input.inputProcessor = stage
        stage.threeInARowGameButton.addClickListener { game.screen = ThreeInARowScreen(game) }
        stage.archeryButton.addClickListener { game.screen = ArcheryScreen(game) }
    }

    override fun render(delta: Float) {
        Gdx.gl.glClearColor(1f, 1f, 1f, 1f)
        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT)


        stage.draw()
    }

    override fun hide() {

    }

    override fun pause() {

    }

    override fun resume() {

    }

    override fun resize(width: Int, height: Int) {

    }

    override fun dispose() {

    }
}